tests/elementsTests/webTables.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { test } from "../baseTest";
+import { expect } from "@playwright/test";
+
+test.describe('Open the main page of the site. Click to the Elements button. The elements page has opened.', () => {
+    test.beforeEach (async ({pageManager})=> {
+    await pageManager.main.Buttons.elementsButton.click();
+    }) 
+
+
+    test ('Click to Web Tables button. Click to the add button. Fill correct values. New member has been added', async ({pageManager,page}) => {
+        const randomFirstName = pageManager.randomValues.getRandomFirstname();
+        const randomLastName = pageManager.randomValues.getRandomLastName();
+        const randomAge = pageManager.randomValues.getRandomAge();
+        const randomEmail = pageManager.randomValues.getRandomEmail();
+        const randomSalary = pageManager.randomValues.getRandomSalary();
+        const randomDepartment = pageManager.randomValues.getRandomDepartment();
+        await addRandomUserToWebTable(pageManager, randomFirstName, randomLastName, randomAge, randomEmail, randomSalary, randomDepartment);
+        await expect (page.locator('div:nth-child(4) > .rt-tr > div').first()).toContainText(randomFirstName);
```

Comment:
In this case, all the expects can be put into a separate function since all checks will be repeated